coalib/settings/ConfigurationGathering.py

```diff
@@ -240,9 +240,15 @@ def load_configuration(arg_list,
             Setting('config', re.escape(find_user_config(os.getcwd()))))
 
     targets = []
+    EXCLUDE_PREFIX = '!'
+    EXCLUDE_LENGTH = len(EXCLUDE_PREFIX)
```

Comment:
nah rather inline it ;)
As imo it should be called `EXCLUDE_PREFIX_LENGTH`, and then `len(EXCLUDE_PREFIX)` is shorter and the same ;)